experiments/AMIP/moist_mpi_earth/slab_ice/slab_init.jl

```diff
@@ -60,3 +60,6 @@ function ice_init(::Type{FT}; tspan, saveat, dt, space, ice_mask, stepper = Eule
 end
 
 get_ice_mask(h_ice, FT) = h_ice > FT(0) ? FT(1) : FT(0)
+
+# file-specific
```

Comment:
could we add a doc string explaining what this is doing?

Reply:
pass in mask so that it is not global?

Reply:
pass in FT also

Reply:
sure :)